CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ x.x.x Release notes (yyyy-MM-dd)
 * Fix a crash when adding a new property to an existing class with over a
   million objects in the Realm.
 * Fix errors when opening encrypted Realm files created with writeCopyToPath.
+* Fix crashes or incorrect results for queries that use relationship equality
+  in cases where the `RLMResults` is kept alive and the target table of the
```

Comment:
... is kept alive when the target class of the relationship has had objects deleted.

We try to avoid table/row in user docs, and the only reason things move is due to deletions.